lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -99,6 +99,11 @@ div.course-wrapper {
     padding: ($baseline*2);
     line-height: 1.6;
 
+    .msg-title {
+        margin-bottom: 10px;
+        font-weight: bold;
```

Comment:
Let's use one of our [font weight mixins](https://github.com/edx/edx-platform/blob/master/lms/static/sass/elements/_typography.scss#L6) here.